pkgs/development/tools/profiling/pyprof2calltree/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonApplication, fetchPypi }:
+
+buildPythonApplication rec {
+  pname = "pyprof2calltree";
+  version = "1.4.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "38a0774f7716d5303d4c57ff89ec24258154942666e4404558f6ac10f8bb1e52";
+  };
+
+  doCheck = false;
```

Comment:
Don't forget to add `checkInputs` and enable tests or mention why they are disabled.

Reply:
I'll look into it a bit more; if I run python setup.py test on a fresh pyprof2calltree clone the tests work fine, but the nix build freaks out.

Reply:
I've seen this error before but only seldom. I think the cause is that `setuptools` runs `build_ext` (for I don't know what reason), "building" the module in place, while it actually already exists. I can't think of another reason that `sys.modules` changes.

I think a workaround is to write
```
${python.interpreter} -m unittest discover -s test
```

Reply:
Thanks. Not quite there yet though (same result in nix-shell and nixpkgs tree):

======================================================================
ERROR: test_integration (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_integration
Traceback (most recent call last):
  File "/nix/store/87l8l9w3x8saxqz9sfcfw2i3jiilmxfi-python-2.7.14/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/nix/store/87l8l9w3x8saxqz9sfcfw2i3jiilmxfi-python-2.7.14/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/tmp/nix-build-pyprof2calltree-1.4.3.drv-0/pyprof2calltree-1.4.3/test/test_integration.py", line 5, in <module>
    from .profile_code import top, expected_output
ValueError: Attempted relative import in non-package

Reply:
That's a valid bug in their `sdist`. They forgot to include `__init__.py` (and also `profile_code.py`).

Reply:
Is `doCheck = false;` (with a comment) an option then? I can raise an issue upstream, but it's not my package.

Reply:
Yes, it is. Raising an issue is a good idea, although I did notice they moved the test files around a bit since the current release.